mlrun/db/httpdb.py

```diff
@@ -2584,6 +2585,7 @@ def create_user_secrets(
         )
 
     @staticmethod
+    @helpers.run_once
```

Comment:
This function is only ran when the httpdb tries to connect to the API which is when:
1. 1st time the client imports mlrun
2. API URL has changed
3. we force reconnection which is when we set the config.dbpath

For 1 and 2 we should always run the validation
For 3 I think we should force reconnection only if the value has changed

My main concern is if the server version has changed and the client will not get warnings even though they should

Reply:
fair points. I will rethink this one alltogether. closing until then